### DISCUSSION TITLE
Rounded corners for medias

### DISCUSSION THREAD

**Discussion by mgogoulos** [mediacms-io/mediacms#1073] [Show and tell]
The latest release adds a number of aesthetic changes, including rounded corners in index page and rounded corner in videos. What do you think?

Check it out on https://demo.mediacms.io/

**Comment by aayusharyan** [comment]
For My Media list. The rounded edges with the text which is just above it looks a bit weird.
Ref:
![image](https://github.com/user-attachments/assets/4ab05ec3-8fa4-43a1-9542-559b81c05b19)

**Reply by mgogoulos** [reply]
it's a little rough there indeed and needs work
